react-ui/src/project/index.js

```diff
@@ -78,28 +80,30 @@ const Project = () => {
             method: "post",
             url: `http://localhost:8000/${metric}`,
             data: {
-                projectslug: project
+                projectslug: project,
```

Comment:
Can you please confirm all the other metrics work fine after adding this change?

Reply:
And add an evidence with the dates being passed.

Reply:
Yes, here is the screenshot:

With the dates added:
![image](https://github.com/sverma89asu/SER516-Team-Cleveland/assets/157744391/fc2e320b-273f-4db5-a8e8-858348a80307)

Without the dates added:
![image](https://github.com/sverma89asu/SER516-Team-Cleveland/assets/157744391/2113ffc1-fb04-468a-9cbf-78f900f065c0)